test/e2e/serial/e2e_serial_test.go

```diff
@@ -45,10 +54,34 @@ func TestSerial(t *testing.T) {
 var _ = BeforeSuite(func() {
 	// this must be the very first thing
 	rand.Seed(time.Now().UnixNano())
+
+	var nrtList nrtv1alpha2.NodeResourceTopologyList
```

Comment:
the code looks correct, but we have plans to re-add these tests in `cnf-tests` (someday...) so this should be wrapped in a helper. Moving this code into `serialconfig` is an option, or we can add another abstraction if needed.

Reply:
I moved the code to a new function in `serialConfig`. Not really proud of the name though